bin/registerServerTopology.sh

```diff
@@ -409,7 +409,7 @@ function request_to_remote() {
   #get response
   IFS=$'\n' REMOTE_API_RESPONSE=($(curl -s -X PATCH ${INSECURE:+--insecure} -i -H "Content-Type: application/json" -H "X-AUTH-TOKEN: ${API_TOKEN}" \
     -d "${REMOTE_PAYLOAD}" \
-    "${API_CURRENT_NODE_PROTOCOL}://${CURRENT_NODE_ADDRESS}:${API_CURRENT_NODE_PORT}/${ROOT_CENTREON_FOLDER}/api/latest/platform" | grep -E "(HTTP/|message)"))
+    "${API_CURRENT_NODE_PROTOCOL}://${CURRENT_NODE_ADDRESS}:${API_CURRENT_NODE_PORT}/${API_CURRENT_NODE_CENTREON_FOLDER}/api/latest/platform" | grep -E "(HTTP/|message)"))
```

Comment:
not related to this PR, but `API_CURRENT_NODE_CENTREON_FOLDER` name seems not appropriate
`API_CURRENT_NODE_BASE_URI` makes more sense
centreon folder can be stored in `/usr/share/my-monitoring-software`, and web ui exposed on `/my-centreon`